runtime-tests/56/goss.yaml

```diff
@@ -0,0 +1,8 @@
+file:
+  /etc/php/5.6/mods-available/newrelic.ini:
```

Comment:
there needs to be a negative test for this (set to false) in the default or base php goss (not dgoss) tests